### ISSUE TITLE
Watchers Tab Error

### ISSUE THREAD

**Issue by MTEngineUser** [Anveio/mturk-engine#9]
# Issue summary

Hi, I discovered this utility a couple of days ago, and I love it, however, the 'Watchers' tab now only shows a blank white page. 

I had 4 or 5 watchers setup (1 or 2 actively running), but now when clicking that tab, the entire page goes blank white, forcing me to refresh the page to get the script to come back. 

After, refreshing the page, resizing the window, and restarting the browser altogether, the 'Watchers' tabs still only shows a blank white page.

The watchers tab and the several watchers were working fine up until a few hours ago.

It seems like a simple display error, except I can't figure out what it could be.

All other tabs (Search, Queue, Blocklist, Account, Settings) work fine.

## Expected behavior

When I click on the 'Watchers' tab, I want to see the watchers that I had setup and running (if I had left them running)

## Actual behavior

When I click on the 'Watchers' tab, the result is a completely blank white page with nothing else on it.

## Steps to Reproduce the Problem

1. Access the url: https://www.mturk.com/mturk/findhits?mturkengine
2. Click on the 'Watchers' tab
3. Blank white page :(

The last thing I remember doing was stopping a watcher after it had caught a HIT, but 2 others watchers were going at that time. It has only shown a blank white page since.

## Specifications

- Mturk Engine version: 1.3.2
- Are you using the Lite version?  No
- Browser: Google Chrome Version 62.0.3202.94 (Official Build) (64-bit)

**Comment by Anveio** [comment]
Hi @MTEngineUser thanks for posting this issue, and I'm sorry to hear that you're running into this bug. A lot of other people have been having this problem and it's been tricky to reproduce but I should have a fix coming in 1.4.0. I believe the problem is caused by watchers writing to your browser's local storage too often and causing it to incompletely stored. So when you refresh the page the data related to your watchers is corrupted, causing the crash.

Unfortunately, I don't have a fix for you other than irreversibly deleting your watchers. If you're willing to do that, here's how:

1. Open Mturk Engine and open your browser's console (F12 or Ctrl + Shift + I)
2. Paste in the following into your console:
```
var x = document.createElement('script')
x.src = 'https://cdnjs.cloudflare.com/ajax/libs/localforage/1.5.5/localforage.js'
document.head.appendChild(x)
```
3. Then paste in the following command into the console:
```
localforage.removeItem('reduxPersist:watchers')
```
4. Refresh your browser window. All of your watchers should now be gone and the watchers tab should no longer cause the app to crash.

To reiterate, **this will delete all of your watchers** and you'll have to add them again manually. Also coming in 1.4.0 (although I don't have an ETA yet) is a way to easily backup and restore all of your data (including watchers) if something like this ever happens again. In the mean time, since the problem seems to be only with watchers, I would recommend turning off all of your watchers before closing the tab or refreshing.

**Comment by MTEngineUser** [comment]
Hi, Anveio. Thanks for the response.

I pasted the code into the console as suggested and it brought back the watchers tab! I didn't have too many of them, so adding them back won't be much of a problem.

I will be sure to turn off all of my watchers before closing/refreshing the tab again in the future and I am looking forward to version 1.4.0 whenever it comes out. Until then, cheers!
